content/blog/أعلان-الفائزين-بجائزة-أسس-للكتابة-في-ش-4/index.ar.md

```
@@ -15,7 +15,7 @@ sourceUrl: "https://aosus.org/1173"
 lang: "ar"
 ---
 
-![](writing-winners-3-2022-1024x576-febaec6b-2.jpeg)
+![](writing-winners-3-2022-1024x576-febaec6b.jpeg)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)